addons/mrp/models/mrp_workcenter.py

```diff
@@ -147,13 +148,18 @@ def _compute_productive_time(self):
         for workcenter in self:
             workcenter.productive_time = count_data.get(workcenter.id, 0.0) / 60.0
 
+    @api.depends('oee_float')
+    def _compute_oee_integer(self):
```

Comment:
you can not use widget="integer" in the view instead of creating a new field?

Reply:
Yes, I can, I just didn't think of it